src/controller/webrtc/WebRTC.cpp

```diff
@@ -0,0 +1,127 @@
+/*
+ *
+ *    Copyright (c) 2025 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#include "WebRTC.h"
+#include "Callbacks.h"
+#include "WebRTCClient.h"
+
+#include <memory>
+#include <thread>
+#include <unistd.h>
+
+#include <rtc/description.hpp>
+#include <rtc/rtc.h>
+#include <rtc/rtc.hpp>
+
+#include <lib/support/logging/CHIPLogging.h>
+
+namespace chip {
+namespace webrtc {
+
+static std::mutex g_mutex;
+static std::map<WebRTCClientHandle, std::shared_ptr<WebRTCClient>> g_clients;
+
+WebRTCClientHandle webrtc_client_create()
```

Comment:
Why this is not a proper C++ class? Matter SDK is a C++ project, do not create C API if not required.

Reply:
Yes, but this is not Python binding, this is still in the matter SDK namespace. You have python binding in `src/controller/python/chip/webrtc/RTCClient.cpp`. Using C in `python/chip/webrtc/RTCClient.cpp` is perfectly fine fore me. However, this C binding should use C++ from matter SDK.